acceptance/bundle/deploy/dashboard/simple_outside_bundle_root/databricks.yml.tmpl

```diff
@@ -0,0 +1,17 @@
+bundle:
+  name: deploy-dashboard-outside-bundle-root-$UNIQUE_NAME
```

Comment:
what's the difference between this test and simple_syncroot? The config seems to be very similar? Can you add a comment here on in the script explaining the purpose of this test and difference from simple_syncroot?